module_skeleton.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import rkl
+import math
+import xarray as xr
+from time_utils import datetime_to_float, datetime_from_float
+import pandas as pd
+
+# interferance and clutter removal
+
+# frequency bank of matched filters for a single pulse
+# mf_rx = 2D array (delay, frequency)
+def freq_dft(array1, array2):
+    y = rkl.delay_multiply.delaymult_like_arg2(array2.values, array1.values/np.sum(array1.values), R=1)
+    z = np.fft.fft(y)
+    global f
```

Comment:
f is included in the DataArray as a coordinate, so it doesn't need to be global. In general, using global variables is asking for bugs.